bring/src/main/java/com/bobocode/petros/bring/scanner/impl/DefaultClassPathBeanDefinitionScanner.java

```diff
@@ -1,15 +1,84 @@
 package com.bobocode.petros.bring.scanner.impl;
 
+import com.bobocode.petros.bring.annotation.Component;
+import com.bobocode.petros.bring.annotation.Repository;
+import com.bobocode.petros.bring.annotation.Service;
 import com.bobocode.petros.bring.context.domain.BeanDefinition;
 import com.bobocode.petros.bring.scanner.ClassPathBeanDefinitionScanner;
+import com.bobocode.petros.bring.utils.BeanNameUtils;
+import lombok.extern.slf4j.Slf4j;
 
+import java.lang.annotation.Annotation;
+import java.util.Arrays;
+import java.util.Collections;
 import java.util.List;
 import java.util.Set;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
 
+import static com.bobocode.petros.bring.context.domain.BeanScope.SINGLETON;
+import static com.bobocode.petros.bring.context.domain.BeanScope.getScopeAsString;
+
+/**
+ * Implementation of {@link DefaultClassPathBeanDefinitionScanner}
+ *
+ * @author YOliinyk
+ * @version 0.0.1
+ */
+@Slf4j
```

Comment:
if you're not using logger then please remove it

Reply:
it would be great to have a logging...